builtin/game/chatcommands.lua

```diff
@@ -15,6 +15,27 @@ function core.register_chatcommand(cmd, def)
 	core.registered_chatcommands[cmd] = def
 end
 
+function core.unregister_chatcommand(name)
+	if not core.registered_chatcommands[name] then
```

Comment:
I'd prefer this to be an `if else end` statement with the condition reversed and the return removed. Less jump and negations (better for mind)